kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/DefaultConfiguration.java

```diff
@@ -33,7 +34,19 @@
 public class DefaultConfiguration {
 
   // initialize the default configuration from the environment
-  private static Configuration conf = new Configuration();
+  private static Configuration conf;
+
+  static{
+    //read system property for Oozie Action Configuration XML
+    String oozieActionXml = System.getProperty("oozie.action.conf.xml");
+    if(oozieActionXml != null && oozieActionXml.length() > 0) {
+      Configuration.addDefaultResource(new Path("file:///", oozieActionXml).toString());
```

Comment:
So this code I borrowed from Oozie's Hive2Main[1].  Do default resources behave differently than the config.addResource(...)

[1] - https://github.com/apache/oozie/blob/5e507124ec5fc06e7723f7b6c589afd09143c513/sharelib/hive2/src/main/java/org/apache/oozie/action/hadoop/Hive2Main.java#L74